README.md

```diff
@@ -1,18 +1,142 @@
-# Cantara
+[Cantara](docs/static/img/cantara.svg 'Cantara Logo')
 
-## Fullstack TypeScript React Apps in Minutes. Stop configuring, start coding.
+<h1 align="center">
+   <a href="https://cantara.js.org">Cantara</a>
+</h1>
 
-[Documentation](/docs/docs/getting_started.md)
+<h3 align="center">
+<span style="color: #305BFF;">C</span>reate <span style="color: #305BFF;">A</span> <span style="color: #305BFF;">N</span>ode, <span style="color: #305BFF;">T</span>ypeScript <span style="color: #305BFF;">A</span>nd <span style="color: #305BFF;">R</span>eact <span style="color: #305BFF;">A</span>pplication
+</h3>
 
-## For maintainers
+<p align="center">
+  <a href="https://github.com/semantic-release/semantic-release">
+    <img src="https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg" alt="semantic-release" />
+  </a>
+  <a href="http://commitizen.github.io/cz-cli/">
+    <img src="https://img.shields.io/badge/commitizen-friendly-brightgreen.svg" alt="Commitizen friendly" />
+  </a>
+  <a href="">
+    <img src="https://github.com/CantaraJS/cantara/workflows/Release/badge.svg" alt="Release" />
+  </a>
+  <a href="https://spectrum.chat/cantara">
+    <img src="https://withspectrum.github.io/badge/badge.svg" alt="Join the community on Spectrum" />
+  </a>
+</p>
 
-Create a .env file in the root of the project.
-Set the following values:
+<h3 align="center">
+  <a href="https://cantara.js.org/docs/quick_start/">Quickstart</a>
+  <span> · </span>
+  <a href="https://cantara.js.org/docs/docs_intro/">Tutorial</a>
+  <span> · </span>
+  <a href="https://cantara.js.org/docs/starter_templates/">Starters</a>
+  <span> · </span>
+  <a href="https://cantara.js.org/docs/maintainers_intro">Contribute</a>
+  <span> · </span>
+  Support: <a href="https://spectrum.chat/cantara">Spectrum</a>
+</h3>
 
+Cantara is a CLI tool to create Fullstack React applications in minutes
+
+- **Zero configuration**: Cantara takes care of all the configurations for your project. No more fiddling around with webpack, typescript, or jest configs. There is even a config for the VS Code debugger.
+- **Testing included**: Have a complete testing-setup before you even wrote your first line of code. [Jest](https://jestjs.io/) and [Cypress](https://www.cypress.io/) allow for all modern practices of automated software testing.
+- **Backends**: Fullstack applications have one or more backends. Build express or serverless APIs that easily connect with the rest of your application.
+- **Mono-repository**: All of your code lives inside a single repository. This makes it easy to share code between all your applications, while still allowing you to treat them as separate packages.
```

Comment:
"while still allowing you to treat them as separate packages." -> "while still keeping a clear separation of concerns thanks to packages."